src/main/java/edu/cornell/kfs/fp/service/RecurringDisbursementVoucherDocumentService.java

```diff
@@ -35,11 +37,33 @@
 	List<RecurringDisbursementVoucherPDPStatus> findPdpStatuses(RecurringDisbursementVoucherDocument recurringDisbursementVoucherDocument);
 	
 	/**
+	 * Cancels all DVs spawned by the Recurring Disbursement Voucher that are in saved status.
+	 * @param recurringDisbursementVoucherDocument
+	 * @param cancelMessage
+	 * @return A list of DV document numbers that have been canceled.
```

Comment:
This method and the one below it are mentioning lists in their Javadocs even though they actually return Sets. Please updated accordingly to avoid confusion.